### PR TITLE
feat(cwe): add cwe into sarif report and KICS CLI results

### PR DESCRIPTION
**Pull request by ArturRibeiro-CX**
**Proposed Changes**

- add cwe into sarif report and KICS CLI results.

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: pkg/report/model/cyclonedx_test.go [hunk 1]
```diff
@@ -24,8 +24,9 @@ var metadata Metadata = Metadata{
 	},
 }
 
+// Update XMLNS version to 1.5 from 1.3
 var initCycloneDxReport CycloneDxReport = CycloneDxReport{
-	XMLNS:        "http://cyclonedx.org/schema/bom/1.3",
+	XMLNS:        "http://cyclonedx.org/schema/bom/1.5",
```
Comment (JoaoCxMartins) [inline pkg/report/model/cyclonedx_test.go:29]: remove this development to another branch

**Review by JoaoCxMartins — CHANGES_REQUESTED** [review]
remove development of cyclonedx to another branch

File: pkg/report/model/cyclonedx.go [hunk 2]
```diff
@@ -218,7 +218,8 @@ func InitCycloneDxReport() *CycloneDxReport {
 			},
 		},
 	}
-
+	// Update cycloneDX version to 1.5 (from 1.3 to 1.5 nothing needed to change since we only create 1 bom at a time,
```
Comment (JoaoCxMartins) [inline pkg/report/model/cyclonedx.go:221]: Remove these comment lines

**Review by JoaoCxMartins — APPROVED** [review]

File: pkg/report/model/sarif.go [hunk 3]
```diff
@@ -43,6 +43,7 @@ type ruleCISMetadata struct {
 type sarifMessage struct {
 	Text              string          `json:"text"`
 	MessageProperties sarifProperties `json:"properties,omitempty"`
+	CWE               string          `json:"cwe,omitempty"`
```
Comment (gabriel-cx) [inline pkg/report/model/sarif.go:46]: do we really need this item?

**Review by gabriel-cx — APPROVED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -66,6 +66,7 @@ RUN wget https://github.com/GoogleCloudPlatform/terraformer/releases/download/0.
 # kics-scan ignore-line
 COPY --from=build_env /app/bin/kics /app/bin/kics
 COPY --from=build_env /app/assets/queries /app/bin/assets/queries
+COPY --from=build_env /app/assets/cwe_csv/* /app/bin/assets/cwe_csv
```
Comment (gabriel-cx) [inline Dockerfile:69]: kindly perform this same update into this [dockerfiles](https://github.com/Checkmarx/kics/tree/master/docker) if needed

**Review by gabriel-cx — CHANGES_REQUESTED** [review]

**Comment by Jeeppler** [comment]
Relates to: https://github.com/Checkmarx/kics/issues/6373

**Review by JoaoCxMartins — DISMISSED** [review]

**Review by cxMiguelSilva — APPROVED** [review]
Awesome work @ArturRibeiro-CX 🙌
Just make sure the dockerfiles work as expected for the ones not tested in workflows as @gabriel-cx  mentioned.

**Review by gabriel-cx — APPROVED** [review]

**Pull request merged**